.github/workflows/release.yaml

```diff
@@ -20,21 +23,124 @@ jobs:
         with:
           go-version: '1.22'
 
+      - name: Build and Package Go executable
+        run: |
+          cd src
+          go mod tidy
+          GOOS=linux GOARCH=amd64 go build -o bootstrap main.go
+          zip -r ../$log_forwarder_zip_file_name .
+          cd ..
+
+      - name: Upload artifact
```

Comment:
Where is this getting uploaded? Why do we need to upload?

Reply:
I am uploading the code to github artifact which is github's own cloud storage. To parallelize lambda code deployment to all the regions s3 buckets, I moved it out as a separate job (github runs entire job parallely and not individual steps inside job). Since every job runs in a different VM we have to share files between jobs using artifacts